benchmark/training/training_benchmark.py

```diff
@@ -0,0 +1,189 @@
+import argparse
+import ast
+
+import torch
+import torch.nn.functional as F
+from tqdm import tqdm
+from utils import get_dataset, get_model
```

Comment:
I got error here.
Traceback (most recent call last):
  File "/home/user/yanbing/pyg/pytorch_geometric/benchmark/training/training_benchmark.py", line 7, in <module>
    from utils import get_dataset, get_model
ModuleNotFoundError: No module named 'utils'